src/mmw/js/src/analyze/controllers.js

```diff
@@ -57,7 +57,8 @@ var AnalyzeController = {
 
         App.state.set({
             'active_page': utils.analyzePageTitle,
-            'was_analyze_visible':  true,
+            'was_analyze_visible': true,
+            'was_compare_visible': false,
```

Comment:
This ensures compare isn't visible when flipping between analyze and model, after having already seen compare.